frappe/commands/site.py

```diff
@@ -496,6 +496,29 @@ def add_system_manager(context, email, first_name, last_name, send_welcome_email
 		raise SiteNotSpecifiedError
 
 
+@click.command("add-user")
+@click.argument("email")
+@click.option("--first-name")
+@click.option("--last-name")
+@click.option("--password")
```

Comment:
Maybe password input could better be done via https://docs.python.org/3/library/getpass.html ?

Reply:
@barredterra We are following the way of add-system-manager bench command which suppose to be the most important role in the system
what do you think?

Reply:
I guess we can follow the established practice here

Reply:
`getpass` won't be friendly in scripts. You need to stream input to stdin 🥴 it gets weird.